### DISCUSSION TITLE
Email & Push notifications are inconsistent

### DISCUSSION THREAD

**Discussion by shalak** [netdata/netdata#14409] [Q&A]
I'm running a pretty fresh installation, didn't mess around that much with it yet. 

I noticed, that some of my alerts/warnings come to email, some other to browser push notifications, and some to both. Why is that?

**Comment by ilyam8** [comment]
I guess it is because email notifications (and other notification methods) can be delayed by [delay](https://learn.netdata.cloud/docs/agent/health/reference#alarm-line-delay) or silenced (`to: silent`), but these settings do not affect browser push notifications.
